src/measures.tex

```diff
@@ -192,38 +195,32 @@ \subsection{Spaces of Measures}
 %  The result follows from Lemma~\ref{theta.limit_y}.
 %\end{proof}
 
-Now fix $0< \xi<1$ and let $x\in\R_{\geq 0}$. For simplicity, denote by $\Z(\!(T)\!)_{r'}$ the group $\Lm_{r'}(\ast)$.
-One of the key results proven in \cite[\S 6]{Analytic} is the surjectivity and continuity of the following map.
+%Now fix $0< \xi<1$ and let $x\in\R_{\geq 0}$. For simplicity, denote by $\Z(\!(T)\!)_{r'}$ the group $\Lm_{r'}(\ast)$.
+%One of the key results proven in \cite[\S 6]{Analytic} is the surjectivity and continuity of the following map.
 
 \begin{definition}
   \label{theta}
   \lean{theta.ϑ}
   \uses{laurent-measures}
   \leanok
-  Fix $0<\xi <1$, and let $\vartheta_{\xi}\colon \Z(\!(T)\!)_r\rightarrow \R$ be the evaluation map
+  Let $0<\xi <1$ be a real number. Denote by $\vartheta_{\xi}\colon \Z(\!(T)\!)_{r'}\rightarrow \R$ the evaluation map
   \[
     \sum a_nT^n\longmapsto \sum a_n\xi^n.
   \]
+induced by $T\mapsto \xi$.
 \end{definition}
 
-In order to prove the surjectivity, we use the following construction.
-\begin{definition}
-  \label{theta-y}
-  \lean{theta-y}
-  \leanok
-  For all $x \in \R$ and $n\in\N$, set
-  \[
-    Y_\xi(x, n)=
-    \begin{cases}
-      x&\text{ if } n = 0 \\
-      Y_\xi(x,n-1) - \left\lfloor \frac{Y_\xi(x,n-1)}{\xi^{n-1}} \right\rfloor \xi^{n-1} & \text{ if } n \ge 1
-    \end{cases}
-  \]
-where $\lfloor \ast \rfloor$ denotes the floor of $\ast$, namely the greatest integer that is less or equal than $\ast$.
-\end{definition}
+One of the key results proven in \cite[\S 6]{Analytic} is the surjectivity and continuity of the map $\vartheta_\xi$, together with a proof that its kernel is principal for every $0<\xi<1$. To state this result, we begin with the following lemma.
+
+% \begin{definition}
+%  \label{theta.y}
+%  \lean{theta.y}
+%  \leanok
+%	, namely the greatest integer that is less or equal than $\ast$.
+%\end{definition}
 
 \begin{lemma}
-  \label{theta-surjective}
+  \label{theta.surjective}
```

Comment:
This breaks the blueprint software. LaTeX labels cannot contain `.` for some reason.